Author: Chris Moore

Disable the building of the eglfs emulator plugin, since we do not want to
use this, but qt seems to want to auto load this if it exists

--- a/src/plugins/platforms/eglfs/deviceintegration/deviceintegration.pro
+++ b/src/plugins/platforms/eglfs/deviceintegration/deviceintegration.pro
@@ -8,8 +8,6 @@
 qtConfig(eglfs_mali): SUBDIRS += eglfs_mali
 qtConfig(eglfs_viv): SUBDIRS += eglfs_viv
 qtConfig(eglfs_viv_wl): SUBDIRS += eglfs_viv_wl
-
-qtConfig(opengl): SUBDIRS += eglfs_emu
 
 eglfs_kms_egldevice.depends = eglfs_kms_support
 eglfs_kms.depends = eglfs_kms_support
